qa/tasks/cbt_performance.py

```diff
@@ -149,4 +153,4 @@ def create_cbt_perf_url(self, ctx, config):
         job_id = ctx.config.get('job_id', None)
         branch = ctx.config.get('branch', None)
 
-        return f'{server}:{grafana_port}/d/{Dash_id}?orgId=1&var-branch_name={branch}&var-job_id_selected={job_id}'
+        return f'{server}:{grafana_port}/d/{Dash_id}?orgId=1&var-branch_name={branch}&var-job_id_selected1={job_id}'
```

Comment:
accidental change?
LGTM otherwise!

nit: Commit title `qa/tasks/cbt_performance:`

Reply:
Did you rename `job_id_selected` to `job_id_selected1` on purpose?

Reply:
> Did you rename `job_id_selected` to `job_id_selected1` on purpose?

Yes, we will have in Grafana link 2 job_ids job_id_selected1\job_id_selected2 for comparing, the first one will be picked by the current job id that run